more/problem-sets-competitive-programming.md

```diff
@@ -12,6 +12,7 @@
 
 * [A Way to Practice Competitive Programming](https://github.com/E869120/Competitive-Programming/blob/master/%5BTutorial%5D%20A%20Way%20to%20Practice%20Competitive%20Programming.pdf) - Masataka Yoneda (PDF)
 * [A2 Online Judge](https://a2oj.netlify.app)
+* [A2Z DSA Sheet for FAANG](https://takeuforward.org/strivers-a2z-dsa-course/strivers-a2z-dsa-course-sheet-2)
```

Comment:
```suggestion
* [Strivers A2Z DSA Course/Sheet](https://takeuforward.org/strivers-a2z-dsa-course/strivers-a2z-dsa-course-sheet-2)
```
Don't invent titles! Use the title given by the resource.
This resource would fit in the interactive tutorials list